### DISCUSSION TITLE
curl: (56) Recv failure: Software caused connection abort, Most of the time it's connection time out

### DISCUSSION THREAD

**Discussion by Majestychen** [AndroidIDEOfficial/AndroidIDE#1435] [Q&A]
I can't download SDK successfully on my phone, sometimes it's curl: (56) Recv failure: Software caused connection abort. 
Most of the time it's connection time out. 
Can I download sdk from my computer alone and copy it to the directory on my phone? 
Thank you!

**Comment by itsaky** [comment]
Here is how you can manually install the build tools :
https://androidide.com/blogs/getting-started/2023/07/17/manually-installing-build-tools-in-androidide/
